Formula/azure-cli-2.rb

```diff
@@ -0,0 +1,75 @@
+require "tmpdir"
+
+class AzureCli2 < Formula
+  include Language::Python::Virtualenv
+
+  desc "Azure CLI 2.0"
+  homepage "https://github.com/Azure/azure-cli"
+  url "https://azurecliprod.blob.core.windows.net/releases/azure-cli_packaged_0.1.5.tar.gz"
+  sha256 "cb65651530544c43343e2c80b0d96a681ad0a07367e288ef9fae261093c497f3"
+  depends_on "python"
+
+  # Apply the 'az component' patch
```

Comment:
Upstream has the ability to update itself through commands such as `az component update`.
According to Homebrew documentation, ["We don’t like tools that upgrade themselves"](https://github.com/Homebrew/brew/blob/master/docs/Acceptable-Formulae.md#we-dont-like-tools-that-upgrade-themselves).
These two patches disable the self-updating functionality whilst still enabling upstream to support it as is desired.